index.html

```diff
@@ -81,7 +81,15 @@
 
             <ul id="nav-links" class="nav__links">
                 <li class="nav__item nav__story">
-                    <a href="#story" class="nav__link">Our Story</a>
+                    <a href="/story" class="nav__link story" id="story-link">Our Story</a>
+                    <script>
+                        (function(){
+                            if (window.location.search === '?souls2thepolls') {
+                                var link = document.getElementById('story-link');
+                                link.setAttribute('href', '/story?souls2thepolls');
+                            }
+                        })();
+                    </script>
```

Comment:
I was thinking that when the user has slow internet connection and wants to get to the "Our Story" page and clicked the link before the script runs, the link will not account for souls2thepolls. Is there a way to solve this non-client side?